halo/registry/keeper/registry.proto

```diff
@@ -23,4 +23,7 @@ message Portal {
   bytes           address            = 2; // Portal contract address
   uint64          deploy_height      = 3; // Height this portal contract was deployed at
   repeated uint64 shard_ids          = 4; // Shard IDs supported by this portal
+  uint64          attest_interval    = 5; // The interval, in blocks, at which validators must attest, even if empty
+  uint64          block_period       = 6; // The block period of the chain deployed to, in milliseconds.
```

Comment:
common practice is to add the unit as a suffix, like `_ms`?

Reply:
Done, thanks.